File: user-service/src/main/java/rs/raf/user_service/domain/entity/AuthorizedPersonel.java
package rs.raf.user_service.domain.entity;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDate;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AuthorizedPersonel {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String firstName;
    private String lastName;
    private LocalDate dateOfBirth;
    private String gender;
    private String email;
    private String phoneNumber;
    private String address;

    @ManyToOne
    @JoinColumn(name = "company_id")
    private Company company;
}

File: user-service-kotlin/src/main/kotlin/pack/userservicekotlin/domain/entities/AuthorizedPersonel.kt
package pack.userservicekotlin.domain.entities

import jakarta.persistence.*
import java.time.LocalDate

@Entity(name = "authorized_personel")
open class AuthorizedPersonel(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    open var id: Long? = null,
    open var firstName: String? = null,
    open var lastName: String? = null,
    open var dateOfBirth: LocalDate? = null,
    open var gender: String? = null,
    open var email: String? = null,
    open var phoneNumber: String? = null,
    open var address: String? = null,
    @ManyToOne
    @JoinColumn(name = "company_id")
    open var company: Company? = null,
)
